code/data_preprocessing/phenotype/gene_annotation.ipynb

```diff
@@ -94,10 +94,10 @@
    "source": [
     "sos run gene_annotation.ipynb annotate_coord \\\n",
     "    --cwd output \\\n",
-    "    --phenoFile data/... \\\n",
-    "    --annotation-gtf reference_data/Homo_sapiens.GRCh38.103.chr.reformatted.gene.ERCC.gtf  \\\n",
+    "    --phenoFile data/MWE.pheno_log2cpm.tsv \\\n",
+    "    --annotation-gtf data/gene.gtf  \\\n",
```

Comment:
can we use our `Homo_sapiens.GRCh38.103.chr.reformatted.gene.ERCC.gtf` for MWE purpose and not upload another GTF?

Reply:
Just tried and it works, reverted.